### PR TITLE
add Russian localization

### PR DESCRIPTION
**Pull request by DjRama**

### CODE REVIEW

**Comment by nminchow** [comment]
Hey, sorry for the delay here. Will test this tomorrow!

File: src/i18n/ru/index.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        deleteMessages: {
+          name: 'delete-expired-events',
+          description: 'удалять уведомления о событиях после окончания события',
+        },
+      },
+      errors: {
+        permissions: `В настоящее время у бота нет разрешений «Отправлять сообщения» и «Просмотр сообщений» для этого канала, поэтому уведомления не могут быть отправлены. Как только разрешения будут включены, повторите эту команду!`,
+      },
+      messages: {
+        success: 'Уведомления о Событиях будут публиковаться на этом канале! Используйте '/{unsub}', чтобы остановить публикацию событий здесь. Используйте команду '/{events}' еще раз, чтобы изменить настройки.',
```
Comment (nminchow) [inline src/i18n/ru/index.ts]: These need to be ticks "`" as there are quotes within it.

File: src/i18n/type-transformer.ts [hunk 2]
```diff
@@ -36,6 +37,10 @@ export const localeMapping: LocaleMapping = {
   [Locale.PortugueseBR]: {
     staticMapping: br,
     locale: 'br',
+  },
+  [Locale.RussianRU]: {
```
Comment (nminchow) [inline src/i18n/type-transformer.ts]: the locale type here looks like its just `Locale.Russian`

Comment (nminchow) [reply src/i18n/type-transformer.ts]: Also, be sure to add `| Locale.Russian` to the supported locale list!

**Review by nminchow — COMMENTED** [review]
Thanks for this PR! Before merge, we'll also need to run `yarn run i18n` to generate the automatic type files!

**Comment by nminchow** [comment]
If you'd rather I make these changes, I'm happy to merge this PR and do the cleanup myself. Let me know!

**Comment by DjRama** [comment]
seems to have corrected all the comments.

**Comment by nminchow** [comment]
> seems to have corrected all the comments.

Hmm, I'm still not seeing the auto-generated file updates I'd expect to see with `yarn run i18n`. Would you like to give me write permission to your branch to make those changes, or should I just pluck them into my own? Happy either way!

**Comment by nminchow** [comment]
hey @DjRama - I'm still happy to get this done! I appreciate all of the work here. Would you like me to update your branch?

**Comment by DjRama** [comment]
@nminchow, do whatever is necessary. I just don't really understand what I need to do!

**Pull request closed without merging**